tendrl/node_agent/objects/definition/node_agent.yaml

```diff
@@ -67,4 +67,34 @@ namespace.node_agent:
       list: nodes/$NodeContext.node.id/Networks
       help: "Global subnet wise network interface"
       value: /networks
+    ClusterAlertCounters:
+      enabled: True
+      attrs:
+        warning_count:
+          help: "Count of warning alerts for the cluster"
+          type: int
+        info_count:
```

Comment:
You probably don't need this.
Just AlertCounter suffices
Same in all counters

Reply:
@shtripat is info count for alert is required ?

Reply:
i removed it